src/core/src/Context.cpp

```diff
@@ -145,6 +215,12 @@ void Context::reloadStage() {
     }
 }
 
+void Context::clearAccessorCache() {
+    if (_pCacheDatabase) {
+        _pCacheDatabase->clearAll();
```

Comment:
Maybe I'm missing something, but I see this code is getting hit and no errors inside `clearAll`, but the `.sqlite` file remains the same size. Is that expected?

Reply:
I did not expect that behavior, but I saw it too and checked the database with the sqlite command tool. All the rows are deleted by this call. I don't know enough about sqlite to say whether the file should be trimmed to or if that requires an additional operation.

Reply:
@kring have you noticed this before in unreal/unity?

Reply:
I haven't noticed it, but haven't looked either. It's not too surprising, I guess.